config/eventhub/config.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2021 The Crossplane Authors.
```

Comment:
```suggestion
Copyright 2022 The Crossplane Authors.
```

Reply:
Apache 2 License does not really require year version bump in the header , but definitely don't mind to do it to keep it fresh :)